spec/index.bs

```diff
@@ -2786,6 +2786,32 @@ interface GPUCommandEncoder {
 GPUCommandEncoder includes GPUObjectBase;
 </script>
 
+{{GPUCommandEncoder}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPUCommandEncoder">
+    : <dfn>\[[state]]</dfn> of type [=encoder state=].
+    ::
+        The current state of the {{GPUCommandEncoder}}, initially set to [=encoder state/open=].
+</dl>
+
+Each {{GPUCommandEncoder}} has a current <dfn>encoder state</dfn> on the [=Content timeline=]
+which may be one of the following:
+
+ - "<dfn for="encoder state">open</dfn>" where the {{GPUCommandEncoder}} is available to begin new operations.
+ - "<dfn for="encoder state">recording a render pass</dfn>" where the {{GPUCommandEncoder}} has an active {{GPURenderPassEncoder}}.
```

Comment:
We should probably settle on consistent "encoding" vs "recording" terms, although it's likely not consistent in the rest of the spec today either.